1.3.0/publish.py

```diff
@@ -0,0 +1,20 @@
+import audb
+
+previous_version = '1.2.0'
+version = '1.3.0'
+build_dir = '../build'
```

Comment:
Here we then also need to update the build folder location
```suggestion
build_dir = './build'
```